packages/ts/generator-typescript-plugin-backbone/src/EndpointProcessor.ts

```diff
@@ -6,16 +7,15 @@ import { OpenAPIV3 } from 'openapi-types';
 import type { ReadonlyDeep } from 'type-fest';
 import type { SourceFile, Statement } from 'typescript';
 import EndpointMethodOperationProcessor, { EndpointMethodOperation } from './EndpointMethodOperationProcessor.js';
-import type { BackbonePluginContext } from './utils.js';
 
 export default class EndpointProcessor {
-  readonly #context: BackbonePluginContext;
+  readonly #context: Plugin;
```

Comment:
should the `context` be renamed to `plugin`?

Reply:
What about `owner`?